src/main/java/ee/buerokratt/ruuter/domain/steps/http/HttpPostStep.java

```diff
@@ -17,6 +17,7 @@ public class HttpPostStep extends HttpStep {
 
     @Override
     protected HttpResponse<String> getHttpRequestResponse(ConfigurationInstance ci) {
+        args.addHeaders(ci);
```

Comment:
Considering that i rewrote the httpPost and get methods, then i think it would be reasonable to wait until that logic can me merged into this branch.

Reply:
Pre-approve by me.